chains/opbnbtestnet/metadata.yaml

```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://opbnb-testnet.bscscan.com/api
+    family: etherscan
+    name: opbnbscan
+    url: https://opbnb-testnet.bscscan.com/
+chainId: 5611
+deployer:
+  name: JunayedMh
+  url: https://github.com/JunayedMh
+displayName: Op Bnb Testnet
+domainId: 5611
+isTestnet: true
+name: opbnbtestnet
+nativeToken:
+  decimals: 18
+  name: Binance
+  symbol: BNB
+protocol: ethereum
+rpcUrls:
+  - http: https://opbnb-testnet-rpc.publicnode.com
+    https: https://opbnb-testnet-rpc.bnbchain.org
```

Comment:
Hey @JunayedMh thank you for your pr to the registry! Can you please fix this line that is causing the CI to fail? It is incorrectly formatted. The `rpcUrls` property accepts a list of `http` properties. You can run the `yarn build` command locally to confirm that the issue has been fixed!